src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -212,15 +213,12 @@ export class OnboardingView extends React.Component<
 
   _renderOnboardingStep() {
     if (!this.props.isLoggedIn) {
-      return (
-        <React.Fragment>
-          <Text style={{ textAlign: "center" }}>
-            Welcome to Raha! Please sign up with your mobile number to accept
-            your invite.
-          </Text>
-          <LogIn navigation={this.props.navigation} />
-        </React.Fragment>
-      );
+      this.props.navigation.replace(RouteName.LogIn, {
```

Comment:
Does back button still work with replace?

Reply:
I used `replace` specifically to make sure that you can't hit the back button to go to onboarding, since that would leave you in the onboarding screen without having logged in.